app/models/recipe.rb

```diff
@@ -0,0 +1,4 @@
+class Recipe < ApplicationRecord
+  belongs_to :user #author
```

Comment:
lets try `belongs_to :author, class_name: 'User'` and see if that works: https://apidock.com/rails/ActiveRecord/Associations/ClassMethods/has_many

I think what we've discussed is allowing the interface to be something like `recipe.author => user` right?

We can setup associations with other models with arbitrary names like

`belongs_to :whatever_model` but we can manually point to model we'll be storing our information in. 

So an "Author" is a "kind of" "User" and we'd represent that in our ActiveRecord relation with:

`belongs_to :author, class_name: 'User`.

Try that change and then play with the models in the console like so:

```
1. rails c
2. user = User.create(name: 'Faust', email: 'faustbubba@bubba.boi')
3. recipe = user.create_recipe
4. recipe.author.name # => should return 'Faust'
```

Reply:
Take a look at this blog post and see if you can get `recipe.author` to work.

https://medium.com/@lushiyun/using-aliasing-associations-for-intuitive-development-reflections-from-a-rails-project-faea6c66f9af